### ISSUE TITLE
Loading epub in parts

### ISSUE THREAD

**Issue by pravid** [futurepress/epub.js#964]
Is it possible to load epub in chunks just like PDF?

**Comment by geek1011** [comment]
If you are talking about loading an unpacked epub (i.e. loaded by the .opf file), it *is* already loaded in chunks. If you are talking about loading a packed epub, this is not possible, as the whole archive needs to be downloaded first.

**Comment by pravid** [comment]
I was hoping if we could load whole epub in chunks just like pdf's are loaded.

**Comment by strattonpress** [comment]
@geek1011 So, if I'm loading an unpacked epub, can I limit the user to just read the first chapter?

**Comment by geek1011** [comment]
Yes, that should probably work. I haven't tested this, but if you return a specific status code for the missing files, you should be able to catch it and do something with it using [displayError](http://epubjs.org/documentation/0.3/#renditioneventdisplayerror).
